config/rbac/role.yaml

```diff
@@ -177,6 +177,7 @@ rules:
   - machineconfigs
   verbs:
   - list
+  - watch
```

Comment:
Why do you need this?

Reply:
Not sure still https://github.com/openshift/windows-machine-config-operator/pull/1279#discussion_r994898616

Reply:
I thought @sebsoto figured this out,

Reply:
@saifshaikh48 please add the above to the commit message. If possible please add a comment here too.

Reply:
`make bundle` kicks back any comments in this file. I added the explanation to where to watch is specified in `ignition.go`